ml-agents/mlagents/trainers/ppo/models.py

```diff
@@ -285,6 +165,6 @@ def create_ppo_optimizer(
             * tf.reduce_mean(tf.dynamic_partition(entropy, self.mask, 2)[1])
         )
 
-        if self.use_curiosity:
-            self.loss += 10 * (0.2 * self.forward_loss + 0.8 * self.inverse_loss)
+    def create_ppo_optimizer(self):
```

Comment:
Is ppo_optimizer like "ATM Machine"? :)

Reply:
Oh yeah, yep :P

Proximal Policy Optimization Optimizer - guess the common name for the algorithm is PPO

Reply:
Not a big deal, just sounded funny in my head :)